### PR TITLE
Fix typos and remove duplicate parts in Certificate Management with kubeadm

### PR DESCRIPTION
**Pull request by PabloCastellano**
Some sentences were duplicate

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -123,8 +121,6 @@ These are advanced topics for users who need to integrate their organization's c
 
 The Kubernetes Certificate Authority does not work out of the box.
 You can configure an external signer such as [cert-manager][cert-manager-issuer], or you can use the built-in signer.
-The built-in signer is part of [`kube-controller-manager`][kcm].
-To activate the built-in signer, you pass the `--cluster-signing-cert-file` and `--cluster-signing-key-file` arguments.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: nit: This wording (without “must”) seems better than the other one. I would keep these lines and omit the ones below.

Comment (Rajakavitha1) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: @PabloCastellano could you please incorporate the changes that @sftim suggested.

Comment (PabloCastellano) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: done

**Comment by sftim** [comment]
@PabloCastellano can you rebase this PR against _master_? It looks like it'll be OK to merge.

**Comment by kbhawkey** [comment]
Hello @PabloCastellano , would you have time to rebase this PR?

**Comment by kbhawkey** [comment]
@PabloCastellano ?

**Comment by jimangel** [comment]
@PabloCastellano I'm closing this PR since it's been open since Jan 11 with no action. Feel free to resolve conflicts and re-open. Thanks!

/close

**Pull request closed without merging**